TeamCode/src/main/java/org/firstinspires/ftc/teamcode/helpers/commands/grabSample.java

```diff
@@ -0,0 +1,29 @@
+package org.firstinspires.ftc.teamcode.helpers.commands;
+
+import com.arcrobotics.ftclib.command.ParallelRaceGroup;
+import com.arcrobotics.ftclib.command.SequentialCommandGroup;
+import com.arcrobotics.ftclib.command.WaitCommand;
+import com.arcrobotics.ftclib.command.WaitUntilCommand;
+
+import org.firstinspires.ftc.teamcode.helpers.commands.wrappers.setClawState;
+import org.firstinspires.ftc.teamcode.subsystems.claw.ClawConfiguration;
+import org.firstinspires.ftc.teamcode.subsystems.claw.ClawSubsystem;
+
+public class grabSample extends SequentialCommandGroup {
+
+    public grabSample(ClawSubsystem claw){
```

Comment:
Why pass `ClawSubsystem` here? It makes the command instantiation longer than it needs to be - just pull the claw here using `VLRSubsystem.getInstance`